addons/one-click-fluffyscratch-login/comframe.js

```diff
@@ -0,0 +1,37 @@
+const FLUFFYSCRATCH_ORIGIN = "https://fluffyscratch.hampton.pw";
+const { projectId } = await (await fetch("consts.json")).json();
+window.parent.postMessage({ oneClickFluffyScratchLogin: true, loaded: true }, FLUFFYSCRATCH_ORIGIN);
+let username;
+chrome.runtime.sendMessage({ contentScriptReady: { url: location.href } }, ({ globalState: { auth } }) => {
```

Comment:
how do you have chrome.runtime in non extension context

Reply:
It is an extension context.

Reply:
how